Sources/Classes/RSEventRepository.m

```diff
@@ -193,6 +202,16 @@ - (void) __initiateSDK {
     });
 }
 
+- (BOOL) isUserHasUpdatedFromPostDMTReleaseVersion {
```

Comment:
This will return true if the user updated the SDK from pre-DMT versions as well right? 

Can we update the naming of the method accordingly ?

Reply:
Sure, will update the method name.